<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>465</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Face Recognition</string>
  </property>
  <widget class="QWidget" name="widget">
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>41</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>63</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>451</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="ProcessImage" name="image" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>280</width>
         <height>210</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>320</width>
         <height>240</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="trainButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>70</height>
            </size>
           </property>
           <property name="text">
            <string>Train</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="recButton">
           <property name="minimumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>80</height>
            </size>
           </property>
           <property name="text">
            <string>Rec</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>25</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="text">
            <string>Your ID:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sun11</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>new</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>new1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>new2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>new3</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="captureButton">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Capture</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="abandonButton">
           <property name="maximumSize">
            <size>
             <width>70</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Discard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="spinBox">
         <property name="value">
          <number>85</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QSlider" name="horizontalSlider">
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>85</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>sendButton</zorder>
   <zorder>textBrowser</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>image</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Load">
    <property name="title">
     <string>&amp;Load</string>
    </property>
    <addaction name="actionDefault"/>
    <addaction name="actionAlt"/>
    <addaction name="actionAlt2"/>
    <addaction name="separator"/>
    <addaction name="actionProfileface"/>
    <addaction name="actionUpperbody"/>
    <addaction name="actionFullbody"/>
    <addaction name="actionFrom"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionFullScreen"/>
    <addaction name="actionNormalSize"/>
   </widget>
   <widget class="QMenu" name="menu_Command">
    <property name="title">
     <string>&amp;Command</string>
    </property>
    <addaction name="actionUnload"/>
    <addaction name="actionRest"/>
   </widget>
   <addaction name="menu_Load"/>
   <addaction name="menu_Command"/>
   <addaction name="menu_View"/>
   <addaction name="menu_About"/>
  </widget>
  <action name="actionDefault">
   <property name="text">
    <string>Default</string>
   </property>
  </action>
  <action name="actionAlt">
   <property name="text">
    <string>Alt</string>
   </property>
  </action>
  <action name="actionAlt2">
   <property name="text">
    <string>Alt2</string>
   </property>
  </action>
  <action name="actionProfileface">
   <property name="text">
    <string>Profileface</string>
   </property>
  </action>
  <action name="actionUpperbody">
   <property name="text">
    <string>Upperbody</string>
   </property>
  </action>
  <action name="actionFullbody">
   <property name="text">
    <string>Fullbody</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="text">
    <string>&amp;FullScreen</string>
   </property>
  </action>
  <action name="actionFrom">
   <property name="text">
    <string>From...</string>
   </property>
  </action>
  <action name="actionNormalSize">
   <property name="text">
    <string>&amp;NormalSize</string>
   </property>
  </action>
  <action name="actionUnload">
   <property name="text">
    <string>&amp;Unload</string>
   </property>
  </action>
  <action name="actionRest">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProcessImage</class>
   <extends>QWidget</extends>
   <header location="global">processImage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
